facebook/build.gradle

```diff
@@ -35,7 +35,7 @@ android {
 }
 
 dependencies {
-    api "com.facebook.android:facebook-login:[13.2.0, 14.0["
+    api "com.facebook.android:facebook-login:13.2.0"
```

Comment:
Revert back to as it is. Set a range of _version_ instead of a fixed version at this time.

Reply:
Done in https://github.com/parse-community/Parse-SDK-Android/pull/1181/commits/fd1d6027334129529b937e704c32e4861239e09a